--- a/metadata.c
+++ b/metadata.c
@@ -844,6 +844,10 @@ GetVideoMetadata(const char *path, const
 			xasprintf(&m.mime, "video/x-matroska");
 		else if( strcmp(ctx->iformat->name, "flv") == 0 )
 			xasprintf(&m.mime, "video/x-flv");
+		else if( strcmp(ctx->iformat->name, "rm") == 0 )  
+			xasprintf(&m.mime, "video/x-pn-realvideo");  
+		else if( strcmp(ctx->iformat->name, "rmvb") == 0 )  
+			xasprintf(&m.mime, "video/x-pn-realvideo");  
 		if( m.mime )
 			goto video_no_dlna;
 
